app/routers/api/files.py

```
@@ -1515,6 +1518,175 @@ def bulk_freeze_files(request: BulkFreezeRequest, db: Annotated[Session, Depends
     )
 
 
+@router.post("/bulk/relocate", response_model=BulkActionResponse)
+def bulk_relocate_files(request: BulkRelocateRequest, db: Annotated[Session, Depends(get_db)]):
```

Ruff (0.15.11)

[warning] Too many branches (15 > 12)

(PLR0912)